### PR TITLE
Extract Packer AMI creation to an isolated project.

### PR DESCRIPTION
**Pull request by deanwilson**
The resources required to create AMIs with packer are quite isolated and
can be treated as a distinct unit of code so it's worth extracting them.

### CODE REVIEW

**Comment by deanwilson** [comment]
We can discuss where we want to build AMIs from in the team meeting today.

File: projects/packer_ami_builder/resources/test/templates/ami_building_policy.tpl [hunk 1]
```diff
@@ -53,11 +31,3 @@ resource "aws_iam_policy" "ami_builder_policy" {
       "Resource" : "*"
```
Comment (joshmyers) [inline projects/packer_ami_builder/resources/test/templates/ami_building_policy.tpl]: I think you can lock this down to a particular VPC, if Packer has it's own.

**Comment by surminus** [comment]
If creating a separate VPC solves this particular conundrum I say go with that.

**Comment by surminus** [comment]
:+1: @alexmuller and I reviewed this together and thought it looked good.

**Pull request merged**